### PR TITLE
fix gui bug on ub20 && update doc

### LINKED ISSUE #294: The GUI is not working on Ubuntu
When launching a study from the GUI on Ubuntu (launched from the executable `antares-xpansion-ui`, nothing happens (no messages, no errors). 

However, the GUI is running normally when launched with the Python launcher using `python antares-xpansion-ui.py`

### PR DESCRIPTION
**Pull request by a-zakir**
Closes #294

### CODE REVIEW

File: .github/workflows/single-file-creation-tgz/action.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ runs:
       LSB_RELEASE_VERSION_SHORT=$(lsb_release --short --release)
       cd _install
       rm -f ./antares-xpansion-launcher*
-      pyinstaller -F ../src/python/launch.py -n antares-xpansion-launcher --add-data "../src/python/config.yaml:." --add-data "./bin/:bin"
+      pyinstaller -F ../src/python/launch.py -n antares-xpansion-launcher.exe --add-data "../src/python/config.yaml:." --add-data "./bin/:bin"
       mv ./dist/antares-xpansion-launcher* .
```
Comment (sgatto) [inline .github/workflows/single-file-creation-tgz/action.yml:25]: I would not use `.exe` for linux version

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -37,9 +37,9 @@ find_program(PYINSTALLER "pyinstaller")
 if (PYINSTALLER)
 
     if (MSVC)
-        install(CODE "execute_process(COMMAND ${PYINSTALLER} -F ${CMAKE_CURRENT_SOURCE_DIR}/python/launch.py -n antares-xpansion-launcher --add-data \"${CMAKE_CURRENT_SOURCE_DIR}/python/config.yaml;.\")" )
+        install(CODE "execute_process(COMMAND ${PYINSTALLER} -F ${CMAKE_CURRENT_SOURCE_DIR}/python/launch.py -n antares-xpansion-launcher.exe --add-data \"${CMAKE_CURRENT_SOURCE_DIR}/python/config.yaml;.\")" )
     else()
-        install(CODE "execute_process(COMMAND ${PYINSTALLER} -F ${CMAKE_CURRENT_SOURCE_DIR}/python/launch.py -n antares-xpansion-launcher --add-data \"${CMAKE_CURRENT_SOURCE_DIR}/python/config.yaml:.\")" )
+        install(CODE "execute_process(COMMAND ${PYINSTALLER} -F ${CMAKE_CURRENT_SOURCE_DIR}/python/launch.py -n antares-xpansion-launcher.exe --add-data \"${CMAKE_CURRENT_SOURCE_DIR}/python/config.yaml:.\")" )
```
Comment (sgatto) [inline src/CMakeLists.txt:42]: no `.exe` for linux

**Review by sgatto — APPROVED** [review]
i would not use `.exe`for linux binaries

**Comment by tbittar** [comment]
I have tested the GUI on Ubuntu and it works on my computer ! Nice !

**Pull request merged**